CMakeLists.txt

```diff
@@ -468,6 +468,36 @@ elseif(MIOPEN_BACKEND STREQUAL "OpenCL")
     set(CPACK_DEBIAN_PACKAGE_CONFLICTS miopen-hip)
     set(CPACK_RPM_PACKAGE_CONFLICTS miopen-hip)
 endif()
+
+# Unpack DB files
+function(unpack_db db_bzip2_file)
+    set(KERNELS_DIR "${CMAKE_SOURCE_DIR}/src/kernels")
+    STRING(REPLACE ".bz2" "" db_file "${db_bzip2_file}")
+    if(EXISTS "${db_file}")
+    else(EXISTS "${db_bzip2_file}")
+        message(STATUS "bunzip2 -k ${db_bzip2_file}")
+        execute_process(
+            COMMAND bunzip2 -k ${db_bzip2_file}
+            WORKING_DIRECTORY ${KERNELS_DIR}
+            RESULT_VARIABLE ret
+            )
+        if(NOT ret EQUAL "0")
+            message(WARNING "(${db_file} could not be extracted, file is empty!")
```

Comment:
(a) Some of the fDB files are large enough to "should-be" hosted via lfs; however, keep them as-is would be easier for review and tracking.
(b) Without `git-lfs` to pull for these files, one will meet this warning for all KDB files:
```
root@ztdh17055:~/MIOpen# ls -ltrah src/kernels/gfx*
-rw-r--r-- 1 root root  41M Mar 21 23:24 src/kernels/gfx1030_36.HIP.fdb.txt
-rw-r--r-- 1 root root  19M Mar 21 23:24 src/kernels/gfx803_36.HIP.fdb.txt
-rw-r--r-- 1 root root  13M Mar 21 23:24 src/kernels/gfx803_36.OpenCL.fdb.txt
-rw-r--r-- 1 root root  19M Mar 21 23:24 src/kernels/gfx803_64.HIP.fdb.txt
-rw-r--r-- 1 root root  13M Mar 21 23:24 src/kernels/gfx803_64.OpenCL.fdb.txt
-rw-r--r-- 1 root root  23M Mar 21 23:24 src/kernels/gfx900_56.HIP.fdb.txt
-rw-r--r-- 1 root root 616K Mar 21 23:24 src/kernels/gfx900_56.OpenCL.fdb.txt
-rw-r--r-- 1 root root 631K Mar 21 23:24 src/kernels/gfx900_64.OpenCL.fdb.txt
-rw-r--r-- 1 root root  19M Mar 21 23:24 src/kernels/gfx900_64.HIP.fdb.txt
-rw-r--r-- 1 root root  37M Mar 21 23:24 src/kernels/gfx906_60.HIP.fdb.txt
-rw-r--r-- 1 root root 4.4M Mar 21 23:24 src/kernels/gfx906_60.OpenCL.fdb.txt
-rw-r--r-- 1 root root  23M Mar 21 23:24 src/kernels/gfx906_64.HIP.fdb.txt
-rw-r--r-- 1 root root 3.2M Mar 21 23:24 src/kernels/gfx906_64.OpenCL.fdb.txt
-rw-r--r-- 1 root root  27M Mar 21 23:24 src/kernels/gfx90878.OpenCL.fdb.txt
-rw-r--r-- 1 root root  134 Mar 21 23:26 src/kernels/gfx1030_36.kdb.bz2
-rw-r--r-- 1 root root 199K Mar 21 23:26 src/kernels/gfx1030_36.db.bz2
-rw-r--r-- 1 root root  134 Mar 21 23:26 src/kernels/gfx900_64.kdb.bz2
-rw-r--r-- 1 root root  134 Mar 21 23:26 src/kernels/gfx900_56.kdb.bz2
-rw-r--r-- 1 root root 1.7M Mar 21 23:26 src/kernels/gfx900_56.db.bz2
-rw-r--r-- 1 root root  134 Mar 21 23:26 src/kernels/gfx906_64.kdb.bz2
-rw-r--r-- 1 root root  134 Mar 21 23:26 src/kernels/gfx906_60.kdb.bz2
-rw-r--r-- 1 root root 4.0M Mar 21 23:26 src/kernels/gfx906_60.db.bz2
-rw-r--r-- 1 root root  81M Mar 21 23:26 src/kernels/gfx90878.HIP.fdb.txt
-rw-r--r-- 1 root root  134 Mar 21 23:26 src/kernels/gfx90878.kdb.bz2
-rw-r--r-- 1 root root  11M Mar 21 23:26 src/kernels/gfx90878.db.bz2
-rw-r--r-- 1 root root  71M Mar 21 23:26 src/kernels/gfx90a68.HIP.fdb.txt
-rw-r--r-- 1 root root  134 Mar 21 23:26 src/kernels/gfx90a68.kdb.bz2
-rw-r--r-- 1 root root 9.5M Mar 21 23:26 src/kernels/gfx90a68.db.bz2
-rw-r--r-- 1 root root  70M Mar 21 23:26 src/kernels/gfx90a6e.HIP.fdb.txt
-rw-r--r-- 1 root root  134 Mar 21 23:26 src/kernels/gfx90a6e.kdb.bz2
-rw-r--r-- 1 root root  11M Mar 21 23:26 src/kernels/gfx90a6e.db.bz2